### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by GravuTrad** [SubtitleEdit/subtitleedit#6186] [General]
Hello.

Is it normal to have the version 3.6.7 out since the 13/08/2022, and no updates proposed by the version 3.6.6 installed?

Thanks

**Comment by niksedk** [comment]
Thx for the info - should be fixed now :)

**Comment by GravuTrad** [comment]
It works thanks.

Isn't there an automatic update system?

**Reply by niksedk** [reply]
SE has no automatic update system. Only the check for updates - and the link to the download page.

**Reply by GravuTrad** [reply]
Sorry i would say an automatic warning update system (linked to github news releases, automatic news report when a check update is done).
Do you do it manually? (news of the update in the software)
Was it a check update problem?

**Reply by nkh0472** [reply]
> Sorry i would say an automatic warning update system (linked to github news releases, automatic news report when a check update is done). Do you do it manually? (news of the update in the software) Was it a check update problem?

For now, you can watch this repo by click "Watch" - "custom" - "Releases". And there is a "check update" option in settings, so I guess the feature you said has be built in.
